core/src/toga/app.py

```diff
@@ -784,22 +785,47 @@ def current_window(self, window: Window):
         self._impl.set_current_window(window)
 
     ######################################################################
-    # Full screen control
+    # Presentation mode controls
     ######################################################################
 
+    # ------------------------Deprecated methods--------------------------
+    # Warnings are disabled as old API tests are still in testbed and warnings will cause error.
```

Comment:
This is what `pytest.warns` is for. It both silences the warning, and validates that a warning *has* been raised.